data/json/monsters/feral_humans.json

```
@@ -58,6 +58,66 @@
       "DROPS_AMMO"
     ]
   },
+  {
+    "id": "mon_feral_sapien_spear",
+    "type": "MONSTER",
+    "name": { "str": "feral sapien" },
+    "description": "In this feral's eyes is more intelligence than in those of most of their kind.  They appear to be holding a spear.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 4,
+    "melee_dice_sides": 6,
+    "melee_cut": 2,
+    "dodge": 3,
+    "armor_bash": 9,
+    "armor_cut": 7,
+    "armor_bullet": 12,
+    "special_attacks": [
+      [ "PARROT", 800 ],
```

```suggestion
      [ "PARROT_AT_DANGER", 0 ],
```
Try the above. You may need to decrease the cooldown for the stretch_attack by a bit to keep it at happening that frequently, since they now have another special "attack".

---

Nope, didn't work, just tested it. They just yell a lot less now.

---

Sigh... They should be yelling about as often as dogs bark.

---

I spawned 12 ferals and kept on walking around them for 7 minutes, only 1 message so far.

---

Ah. I see why not. I'm putting up an issue.